libraries/AP_Motors/AP_MotorsSingle.cpp

```diff
@@ -186,7 +178,7 @@ void AP_MotorsSingle::output_armed_stabilizing()
         }
     }
 
-    actuator_allowed = 1.0f - rp_scale * rp_thrust_max;
+    const float actuator_allowed = 1.0f - rp_scale * rp_thrust_max;     // amount of yaw we can fit in
```

Comment:
I have not set the semi-colon and the number of whitespace of the beginning "//" of the comment to 2. I think that it is better to unify.